File: C#/RefactoringWorkshop/RefactoringWorkshop/Bloaters/Dataclumps/Customer.cs
namespace RefactoringWorkshop.Bloaters.Dataclumps
{
    public class Customer
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Title { get; set; }

        public string House { get; set; }

        public string Steet { get; set; }

        public string City { get; set; }

        public string Postcode { get; set; }

        public string Country { get; set; }

        public string AddressSummary() {
            return House + ", " + Steet + ", " + City + ", " + Postcode + ", " + Country;
        }

    }
}

File: Java/RefactoringWorkshop/src/main/java/bloaters/dataclumps/Customer.java
package bloaters.dataclumps;

public class Customer {
    private String firstName;
    private String lastName;
    private String title;
    private String house;
    private String steet;
    private String city;
    private String postcode;
    private String country;

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getHouse() {
        return house;
    }

    public void setHouse(String house) {
        this.house = house;
    }

    public String getSteet() {
        return steet;
    }

    public void setSteet(String steet) {
        this.steet = steet;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getPostcode() {
        return postcode;
    }

    public void setPostcode(String postcode) {
        this.postcode = postcode;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getAddressSummary() {
        return getHouse() + ", " + getSteet() + ", " + getCity() + ", " +
                getPostcode() + ", " + getCountry();
    }
}
